spec/fixtures/site/_data/members.tsv

```diff
@@ -0,0 +1,3 @@
+name	age	sex	location
```

Comment:
`csv` and `tsv` (supported by Jekyll 3.5 and beyond) have been included here for better futureproof